react_ui/src/components/map_data_points/MapDataPointsMap.tsx

```diff
@@ -103,6 +103,20 @@ export default class MapDataPointsMap extends React.Component<
       });
     return this.mapLayer;
   }
+
+  getGCMLayer() {
+    const gcmLayer = L.tileLayer.wms(
+      "https://geo.dev.ecosystem-urbanage.eu/geoserver/GCI/wms?CQL_FILTER=simulationid+=+%27baseline%27",
```

Comment:
This is intended to be hardcoded?

Reply:
yes, it's for the elderly feedback session